src/plugins/unified_search/public/typeahead/suggestions_component.tsx

```diff
@@ -6,22 +6,22 @@
  * Side Public License, v 1.
  */
 
-import { isEmpty } from 'lodash';
 import React, { PureComponent, ReactNode } from 'react';
+import { isEmpty } from 'lodash';
 import classNames from 'classnames';
+// eslint-disable-next-line @kbn/eslint/module_migration
```

Comment:
- [ ] i think we should address that issue 
<img width="654" alt="image" src="https://user-images.githubusercontent.com/20072247/159213878-8dddc7dc-e70e-44c4-bd13-a4f50b591738.png">